docs/3.api/5.kit/11.nitro.md

```diff
@@ -395,3 +395,43 @@ export default defineNuxtModule({
   }
 })
 ```
+
+## `addServerScanDir`
+
+Add directories to be scanned by Nitro. It will check for:
+
+- `api/` folder (which will create API routes)
+- `middleware/` folder (which will register server middleware)
```

Comment:
Since we are not listing all dirs, would be better to only list most useful ones `api/` and `utils/` (maybe `/routes` too, I guess `public/` should work too!)

There are more like `modules/`, `tasks/`, `routes/`, etc which I wish we had a page in nitro we could link to. For `/server/middleware` I am trying to reduce encouragement of it's use and since it is new and specially targeting programmatic users, probably better to not list in more places.

Reply:
Would be nice to add the list of scanner dirs in Nitro docs so we could link from it

Reply:
PR welcome to add the page 👍🏼

Reply:
Should it be a section on https://nitro.unjs.io/guide/auto-imports ?

Reply:
I guess we can make a new `guide/directory-structure/` section for supported scan dirs similar to nuxt..

Reply:
Sure. Let's cut this down and add a link later.